inc/commoninjectionlib.class.php

```diff
@@ -1015,9 +994,8 @@ private function reformatFirstPass()
 
             foreach ($data as $field => $value) {
                 if ($value && $value == "NULL") {
-                    // TODO: fix this code
-                    // if (isset($option['datatype']) && self::isFieldADropdown($option['displaytype'])) {
-                    if (false) {
+                    // TODO: fix this code 
+                    if (false) { // @phpstan-ignore-line
```

Comment:
Clearly, the whole foreach is useless? In that case, deleting it completely (or commenting out the block) would have the same behaviour.